proposals/0017-priced-compute-units.md

```diff
@@ -0,0 +1,34 @@
+---
+simd: '0017'
+title: Priced Compute Units
+authors:
+  - Anoushk Kharangate (SuperteamDAO)
+  - 7Layer (Overclock Validator)
+  - Dubbelosix (Overclock Validator)
+category: Fees
+type: Fees
+status: Draft
+created: 2023-01-05
+---
+
+## Problem
+According to the discussion on the following issue:
+https://github.com/solana-labs/solana/issues/29492
+
+Recently blocks have been reaching their compute unit limit of 48 million CU very frequently. The concerning part is most of these transactions
+are requesting extra compute units without actually needing them; most of these being mev traders. The reason this happens is because
+using extra compute units comes at no extra cost unlike something like rent but validators have to bear the load and aren't compensated for it
+hence a new economic model is required for using compute that disincentivizes extra CU unless absolutely necessary.
+
+## Solution
+Introduce a new price and economic model for compute units. It's important to note that the new model might not want to make it more expensive to vote.
+
+A good way to normalize and do this would be charge a fee per compute unit of 5 x 10<sup>-6</sup> SOL/(vote CU) per CU<br/>
+Example: Txn A requires 10000 CU -> Cost of txn is 10000 x (5 x 10<sup>-6</sup> SOL)/2100 CU  (2100 is the vote CU)
+
+**Note: This would cause problems for CLOBs since doing a large cancel order consumes 20k CU and would cost 0.1 SOL**
+
```

Comment:
The total compute unit based fee would be tied to the existing compute budget program requests that priority fees use. The total fee would be equivalent to the fee per cu times the compute requested. It may be useful to have a lower and upper bound in the compute budget program requests.